plugins/xrandr/msd-xrandr-manager.c

```diff
@@ -1777,8 +1755,8 @@ make_menu_item_for_output_title (MsdXrandrManager *manager, MateRROutputInfo *ou
         /*Load the icon unless the user has icons in menus turned off*/
         icon_settings = g_settings_new ("org.mate.interface");
         if (g_settings_get_boolean (icon_settings, "menus-have-icons")){
-            gtk_container_add (GTK_CONTAINER (box), image);
-            }
+                gtk_container_add (GTK_CONTAINER (box), image);
+        }
```

Comment:
If you do not make any changes to the code, please do not format it, as you will increase the number of lines to review.